### ISSUE TITLE
legacy support adding weird dir to public

### ISSUE THREAD

**Issue by edeustace** [PieLabs/pie-cli#77]
Legacy support is adding assets here:
```
docs/demo/public/_/_/_/_/controllers/node_modules/corespring-legacy-component-dependencies

```
why?

**Comment by edeustace** [comment]
This appears to happen because we have multiple dependencies trying to emit these files.. but only one dependency matches the `context`. For any dependency that doesn't match this it looks like the file loader defaults to `__` as the path.

To fix this (and most likely #78 too), we'll need to polish the webpack config, here is a gist of a working config: https://gist.github.com/edeustace/4fe67d17f9c2a6b11e471bcd1d3d5bb3

**Comment by edeustace** [comment]
The fact that `controllers` is in the path is raised in #79 .
